src/main/java/com/google/api/gax/grpc/InstantiatingChannelProvider.java

```diff
@@ -116,8 +115,8 @@ private ManagedChannel createChannel(Executor executor) throws IOException {
     interceptors.add(new ClientAuthInterceptor(credentialsProvider.getCredentials(), executor));
     interceptors.add(new HeaderInterceptor(serviceHeader()));
 
-    return NettyChannelBuilder.forAddress(serviceAddress, port)
-        .negotiationType(NegotiationType.TLS)
+    //By default ManagedChannel uses TLS negotiationType
```

Comment:
I don't think you need the comment here for something that's completely absent in the final product - this is only information for the change you are making.